packages/components/src/components/files/FileTree.tsx

```diff
@@ -203,18 +208,31 @@ export class FileTree extends PureComponent<FileTreeProps, FileTreeState> {
         loadData()
             .then(data => {
                 let loadedData = data;
+                let permissions = {
+                    canDelete: false,
+                    canEdit: true,
+                    canRead: true,
+                    canRename: false,
+                    canUpload: false
+                };
```

Comment:
If we are going to leave in this hardcoded permissions set for the root node, we should add a comment as to why some of them are set to true by default.